src/components/post/dto/postable.dto.ts

```diff
@@ -10,19 +10,22 @@ import { Post } from './post.dto';
   description: stripIndent`
     An object that can be used to enable Post discussions on a Node.
   `,
+  implements: Resource,
 })
-export abstract class Postable {
+export abstract class Postable extends Resource {
```

Comment:
I'd like to avoid this if possible. Why? Idk just seems like an unrelated change. Do you have a reason or were you just matching comments?